lib/utils/otel/trace.ts

```diff
@@ -0,0 +1,28 @@
+import { Resource } from '@opentelemetry/resources';
+import { BasicTracerProvider, BatchSpanProcessor } from '@opentelemetry/sdk-trace-base';
+import { SEMRESATTRS_SERVICE_NAME } from '@opentelemetry/semantic-conventions';
+import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http';
+
+const provider = new BasicTracerProvider({
+    resource: new Resource({
+        [SEMRESATTRS_SERVICE_NAME]: 'rsshub',
+    }),
+});
+
+const exporter = new OTLPTraceExporter({
+    // optional OTEL_EXPORTER_OTLP_ENDPOINT=https://localhost:4318
```

Comment:
I think this can be updated thru env variable, isn't it?

Reply:
> I think this can be updated thru env variable, isn't it?

due to https://www.npmjs.com/package/@opentelemetry/exporter-trace-otlp-http 
this env would be auto load by this package